src/qt/forms/receiverequestdialog.ui

```diff
@@ -6,68 +6,233 @@
+     </property>
+    </widget>
+   </item>
+   <item row="4" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
+    <widget class="QLabel" name="amount_tag">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Amount:</string>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::NoTextInteraction</set>
+     </property>
+    </widget>
+   </item>
+   <item row="4" column="1" alignment="Qt::AlignTop">
+    <widget class="QLabel" name="amount_content">
+     <property name="text">
+      <string notr="true">0.00000000 BTC</string>
+     </property>
+     <property name="textFormat">
+      <enum>Qt::PlainText</enum>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::TextSelectableByMouse</set>
+     </property>
+    </widget>
+   </item>
+   <item row="5" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
+    <widget class="QLabel" name="label_tag">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Label:</string>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::NoTextInteraction</set>
+     </property>
+    </widget>
+   </item>
+   <item row="5" column="1" alignment="Qt::AlignTop">
+    <widget class="QLabel" name="label_content">
+     <property name="text">
+      <string notr="true">label content</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
-     <property name="alignment">
-      <set>Qt::AlignCenter</set>
+     <property name="wordWrap">
+      <bool>true</bool>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::TextSelectableByMouse</set>
+     </property>
+    </widget>
+   </item>
+   <item row="6" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
+    <widget class="QLabel" name="message_tag">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Message:</string>
+     </property>
+     <property name="textInteractionFlags">
+      <set>Qt::NoTextInteraction</set>
+     </property>
+    </widget>
+   </item>
+   <item row="6" column="1" alignment="Qt::AlignTop">
+    <widget class="QLabel" name="message_content">
+     <property name="text">
+      <string notr="true">message content</string>
+     </property>
+     <property name="textFormat">
+      <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
+     <property name="textInteractionFlags">
+      <set>Qt::TextSelectableByMouse</set>
+     </property>
     </widget>
    </item>
-   <item>
-    <widget class="QTextEdit" name="outUri">
```

Comment:
The QTextEdit widget made it more obvious to users they could select...

Reply:
Since [`QTextEdit`](https://doc.qt.io/qt-5/qtextedit.html) inherits [`QAbstractScrollArea`](https://doc.qt.io/qt-5/qabstractscrollarea.html) with very loose size policy, it is the point of this PR to get rid of it.

Reply:
... then Concept NACK I guess

Without QTextEdit, you won't be able to select and copy everything together.

Reply:
> Without QTextEdit, you won't be able to select and copy everything together.

Maybe a button "Copy ALL" could help?

Reply:
Sounds like a hack

Reply:
There's a copy-URI button which copies the URI. You can tripple click on the URI itself to select it.

The `QTextEdit` used to cover all fields. Tripple click in that edit field wouldn't select the URI, but everything, which doesn't seem useful.